tests/derive.rs

```diff
@@ -257,7 +254,7 @@ mod tests {
 
                 #[derive(ArrayType, Default)]
                 struct FooBar {
-                    foo: bool,
+                    fuu: bool,
```

Comment:
Just curious, why was this changed?

Reply:
Clippy started complaining about it in CI:
https://github.com/mbrobbel/narrow/actions/runs/10737811822/job/29780070914

Reply:
Super pedantic. I ❤️ Clippy.